src/tests/external_address_test.ts

```diff
@@ -416,23 +440,23 @@ describe("ExternalAddress", () => {
     it("should return the stakeable balance successfully", async () => {
       Coinbase.apiClients.stake!.getStakingContext = mockReturnValue(STAKING_CONTEXT_MODEL);
```

Comment:
Side-question since this already existed, but how are we asserting that this is called with the correct request (e.g. being passed the network ID and the primary denomination, etc...)?

Reply:
Ah yeah true, there aren't any assertions there. Added them now for the expected values in the staking context call.